### PR TITLE
Fixes for Twitter Activity channels

### PR DESCRIPTION
**Pull request by rowanseymour**
* Remove a channel even if deactivating it throws an exception
* Remove no longer used `notify_mage` param for `Channel.release` - Mage is notified based on `settings.IS_PROD`
* Fix channel type in Twitter Activity API handler
* Update endpoint used to subscribe to a Twitter Activity API webhook (this appears to have changed)

### CODE REVIEW

File: temba/channels/handlers.py [hunk 1]
```diff
@@ -2922,7 +2922,7 @@ def get(self, request, *args, **kwargs):
         crc_token = request.GET['crc_token']
 
         channel = Channel.objects.filter(uuid=kwargs['uuid'], is_active=True,
-                                         channel_type='TT').exclude(org=None).first()
+                                         channel_type='TWT').exclude(org=None).first()
```
Comment (rowanseymour) [inline temba/channels/handlers.py]: This is the problem with codes that aren't constants. I didn't move handlers into the new channel type packages because in theory they're being replaced soon by Courier - but might still be worth it so that they have access to the channel type.

File: temba/utils/management/commands/test_db.py [hunk 2]
```diff
@@ -166,7 +166,7 @@ def handle_generate(self, num_orgs, num_contacts, seed):
         r.flushdb()
         self._log(self.style.SUCCESS("OK") + '\n')
 
-        superuser = User.objects.create_superuser("root", "root@example.com", "password")
+        superuser = User.objects.create_superuser("root", "root@example.com", USER_PASSWORD)
```
Comment (rowanseymour) [inline temba/utils/management/commands/test_db.py]: Unrelated change so that all the passwords generated by test_db are the same

File: temba/channels/handlers.py (hunk 1 above)
Comment (nicpottier) [inline temba/channels/handlers.py]: How has that ever worked?

Comment (rowanseymour) [inline temba/channels/handlers.py]: I had previously implemented Twitter Activity stuff as a special case of TT channels but then later decided it deserved to be it's own type
